### PR TITLE
N14: Add Nox Futura and Bracket-lib

### PR DESCRIPTION
**Pull request by thebracket**
Adds a graphic and section for Nox Futura.
Adds a small section for bracket-lib, mentioning that its in "stability mode" because it is referenced heavily in my upcoming Rust gamedev book.

I've not made many PRs before, hope I did this right!

_Part of #278_

### CODE REVIEW

File: content/posts/newsletter-014/index.md [hunk 1]
```diff
@@ -123,6 +123,30 @@ Imgur screenshot [gallery].
 [Mimas]: https://github.com/est31/mimas
 [gallery]: https://imgur.com/a/vvo7len
 
+### [Nox Futura]
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-014/index.md:126]: The corresponding URL for this link is missing (might be a copy and paste from an earlier newsletter?)

File: content/posts/newsletter-014/index.md [hunk 2]
```diff
@@ -254,6 +278,18 @@ including a tutorial on how to get started.
 
 ## Library & Tooling Updates
 
+### [Bracket-Lib]
+
+[bracket-lib] (previously `rltk_rs`) by [@blackfuture]
+is a Rust implementation of [C++ Roguelike Toolkit][rltk-cpp].
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-014/index.md:284]: The URLs for these links are also missing.

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
Content looks good to me :+1: There's just a couple of issues with the links.

File: content/posts/newsletter-014/index.md (hunk 2 above)
Comment (thebracket) [reply content/posts/newsletter-014/index.md:284]: The link issues you found should be resolved. Sorry about that!

**Review by 17cupsofcoffee — APPROVED** [review]

**Comment by thebracket** [comment]
Not a problem. I've spent the last couple of months (in between being really busy IRL) getting the new engine structure up to parity with where it was previously, so that makes perfect sense!

**Review by ozkriff — APPROVED** [review]

**Comment by ozkriff** [comment]
Merged, thanks for the PR! Good luck with your new book :)

**Pull request merged**